src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -41,8 +41,10 @@ namespace Windows.UI.Xaml.Controls
 	public partial class ItemsControl : Control, IItemsControl
 	{
 		private ItemsPresenter _itemsPresenter;
-		private SerialDisposable _notifyCollectionChanged = new SerialDisposable();
+
+		private readonly SerialDisposable _notifyCollectionChanged = new SerialDisposable();
 		private readonly SerialDisposable _notifyCollectionGroupsChanged = new SerialDisposable();
+		private readonly SerialDisposable _cvsViewChanged = new SerialDisposable();
```

Comment:
As it's  a control there is no right place to dispose it ... but are you aware that this is not disposed ?

Reply:
The point is not global dispose there, it's unregistering previous.